src/unix/syscall.c

```diff
@@ -263,19 +263,28 @@ static inline tuple resolve_cstring(tuple cwd, const char *f)
 
     buffer a = little_stack_buffer(NAME_MAX);
     char y;
-    while ((y = *f++)) {
+    int nbytes;
+
+    while ((y = *f))
+    {
```

Comment:
The opening brace should be after the while() clause in the preceding line.

Reply:
Right! Sorry, I'm still getting the coding style from my last project out of my system!